src/ServiceControlInstaller.Engine/Instances/MonitoringTransports.cs

```diff
@@ -8,50 +8,102 @@ public class MonitoringTransports
+                SampleConnectionString = "DefaultEndpointsProtocol=[http|https];AccountName=<MyAccountName>;AccountKey=<MyAccountKey>",
+                Matches = name => name.Equals("AzureStorageQueue", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("NServiceBus.AzureStorageQueueTransport, NServiceBus.Azure.Transports.WindowsAzureStorageQueues", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("ServiceControl.Transports.AzureStorageQueues.ServiceControlAzureStorageQueueTransport, ServiceControl.Transports.AzureStorageQueues", StringComparison.OrdinalIgnoreCase)
             },
             new TransportInfo
             {
                 Name = "MSMQ",
                 ZipName = "Msmq",
-                TypeName = "NServiceBus.MsmqTransport, NServiceBus.Core",
+                TypeName = "NServiceBus.MsmqTransport, NServiceBus.Transport.Msmq",
                 SampleConnectionString = string.Empty,
-                Default = true
+                Default = true,
+                Matches = name => name.Equals("MSMQ", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("NServiceBus.MsmqTransport, NServiceBus.Core", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("NServiceBus.MsmqTransport, NServiceBus.Transport.Msmq", StringComparison.OrdinalIgnoreCase)
             },
             new TransportInfo
             {
                 Name = "SQLServer",
                 ZipName = "SqlServer",
-                TypeName = "NServiceBus.SqlServerTransport, NServiceBus.Transport.SQLServer",
+                TypeName = "ServiceControl.Transports.SQLServer.ServiceControlSQLServerTransport, ServiceControl.Transports.SQLServer",
                 SampleConnectionString = "Data Source=<SQLInstance>;Initial Catalog=nservicebus;Integrated Security=True",
-                Help = "When integrated authentication is specified in the SQL connection string the the current installing user is used to create the required SQL tables structure not the service account."
+                Help = "When integrated authentication is specified in the SQL connection string the the current installing user is used to create the required SQL tables structure not the service account.",
+                Matches = name => name.Equals("SQLServer", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("ServiceControl.Transports.SQLServer.ServiceControlSQLServerTransport, ServiceControl.Transports.SQLServer", StringComparison.OrdinalIgnoreCase)
+                          || name.Equals("NServiceBus.SqlServerTransport, NServiceBus.Transport.SQLServer", StringComparison.OrdinalIgnoreCase)
             },
             new TransportInfo
             {
-                Name = "RabbitMQ",
+                Name = "RabbitMQ - Conventional Routing Topology",
```

Comment:
Should we just drop the " - Conventional routing topology" since this is to go to topology going forwards? cc @bording

Reply:
It would align with docs since they only talk about "RabbitMQ" https://docs.particular.net/servicecontrol/monitoring-instances/installation/

Reply:
FYI Service Control uses those names for now. I think the intention here was to align how SC presents them